src/app/tests/suites/certification/Test_TC_OO_2_3.yaml

```diff
@@ -393,8 +397,8 @@ tests:
       PICS: OO.S.A4001 && OO.S.C42.Rsp
       response:
           constraints:
-              minValue: 255
-              maxValue: 345
+              minValue: 300 - PIXIT.OO.MaxCommunicationTurnaround
```

Comment:
In 10d the OnTime countdown starts.
In first 10e readAttribute of OnOff is performed that can take up to PIXIT.OO.MaxCommunicationTurnaround.
In second 10e readAttribute of OnTime is performed that can take up to PIXIT.OO.MaxCommunicationTurnaround.
So in total I think it should be 300 - 2 * PIXIT.OO.MaxCommunicationTurnaround

Reply:
Updated.